packages/console/src/components/ErrorBoundary/index.module.scss

```diff
@@ -0,0 +1,34 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  overflow-y: auto;
+  height: 100vh;
+
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--color-layer-1);
+    color: var(--color-text);
+    padding: _.unit(6);
+    width: 858px;
+    min-height: 100%;
+    margin: 0 auto;
+
+    > *:not(:first-child) {
+      margin-top: _.unit(6);
+    }
+
+    img {
+      height: 300px;
+    }
+
+    img,
+    h2 {
+      margin: 0 auto;
```

Comment:
if we set `align-items: center`, this would be redundant?

Reply:
The exception call stack (`.details`) needs to be left aligned.

Reply:
Add `align-self` to this one?.

Reply:
OK, I'll use `align-self: center` instead of `margin: 0 auto` in this class